db/migrate/20220325174235_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
+      t.integer :commentsCounter
+      t.integer :likesCounter
```

Comment:
- [ ] Please update your naming according to the naming convention your `commentsCounter` should be `comments_counter` and your `likesCounter` should be `likes_counter`

![image](https://user-images.githubusercontent.com/75126481/160175354-0ccb7efb-1b4f-46d4-ad2b-fd7c9a807d82.png)

Reply:
Thank you for the suggestion. I will work on it.